src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,53 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    // Test Case 구현
+    @Test
+    @DisplayName("Set의 크기 확인")
+    void checkSize(){
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
+        assertEquals(3, size);
```

Comment:
다음과 같이 한 문장으로 크기를 테스트할 수 있을 것 같아요 :)
```suggestion
        assertThat(numbers).hasSize(3);
```